src/box/lua/net_box.lua

```diff
@@ -1059,15 +1095,15 @@ this_module.self = {
         end
         local result
         if obj ~= nil then
-            return handle_eval_result(pcall(proc, obj, ...))
+            return handle_eval_result(pcall(proc, obj, unpack(args)))
```

Comment:
I don't think it makes sense. We especially added one extra MessagePack Array on the first level of CALL/EVAL result to support Lua multireturn:

```
tarantool> conn = require('net.box').connect(box.cfg.listen)
---
...

tarantool> function echo(...) return ... end
---
...

tarantool> conn:call("echo", {1, 2, 3})
---
- 1
- 2
- 3
tarantool> conn:eval("return echo(...)", {1, 2, 3})
---
- 1
- 2
- 3
...
```